### ISSUE TITLE
mvn package issue

### ISSUE THREAD

**Issue by maaariii** [cooperlyt/keycloak-phone-provider#61]
When I put "mvn package" in home directory of project, I face following error, Can you please help me

"[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] keycloak-phone-provider                                            [pom]
[INFO] keycloak-phone-provider                                            [jar]
[INFO] keycloak-phone-provider.resources                                  [jar]
[INFO] keycloak-sms-provider-twilio                                       [jar]
[INFO] 
[INFO] ------------< cc.coopersoft:keycloak-phone-provider-parent >------------
[INFO] Building keycloak-phone-provider 2.3.4-snapshot                    [1/4]
[INFO] --------------------------------[ pom ]---------------------------------
[WARNING] The POM for org.apache.maven.plugins:maven-resources-plugin:jar:2.10.0 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for keycloak-phone-provider 2.3.4-snapshot:
[INFO] 
[INFO] keycloak-phone-provider ............................ FAILURE [  0.013 s]
[INFO] keycloak-phone-provider ............................ SKIPPED
[INFO] keycloak-phone-provider.resources .................. SKIPPED
[INFO] keycloak-sms-provider-twilio ....................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.097 s
[INFO] Finished at: 2023-06-27T16:05:54+05:30
[INFO] ------------------------------------------------------------------------
[ERROR] Plugin org.apache.maven.plugins:maven-resources-plugin:2.10.0 or one of its dependencies could not be resolved: Failure to find org.apache.maven.plugins:maven-resources-plugin:jar:2.10.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginResolutionException
"

**Comment by cooperlyt** [comment]
check your maven version .  In my is Apache Maven 3.8.6

**Comment by maaariii** [comment]
What is your java version.My java version is 11.0.19. and maven version is 3.6.3

**Comment by maaariii** [comment]
It's work for me after putting following changes mentioned below in attached images
![Screenshot from 2023-06-27 19-33-16](https://github.com/cooperlyt/keycloak-phone-provider/assets/109349168/eded4aa7-637e-4ccf-80ae-97e637b81b9a)
![Screenshot from 2023-06-27 19-32-56](https://github.com/cooperlyt/keycloak-phone-provider/assets/109349168/2a1c570b-9bb9-4f0f-8256-30d25a546c0c)

**Comment by cooperlyt** [comment]
keycloak 21.X.X  need java 17 +

**Comment by maaariii** [comment]
> keycloak 21.X.X need java 17 +

Yes, understood. But I've faced another Issue in DefaultPhoneVerificationCodeProvider file. So, I comment below code. Please find it in attachment.
![Screenshot from 2023-06-27 19-33-16](https://github.com/cooperlyt/keycloak-phone-provider/assets/109349168/53a603f8-8cb9-4cdd-b487-c41386962b5f)

**Comment by maaariii** [comment]
> > keycloak 21.X.X need java 17 +
> 
> Yes, understood. But I've faced another Issue in DefaultPhoneVerificationCodeProvider file. So, I comment below code. Please find it in attachment. ![Screenshot from 2023-06-27 19-33-16](https://user-images.githubusercontent.com/109349168/249185472-53a603f8-8cb9-4cdd-b487-c41386962b5f.png)

![Screenshot from 2023-06-27 19-32-56](https://github.com/cooperlyt/keycloak-phone-provider/assets/109349168/0c7ed23d-c62c-493c-aa33-04d45a0384b1)
